lib/options/uefi-setup-menu.robot

```diff
@@ -59,3 +59,11 @@ Get UEFI Option
 
     ${state}=    Get Option State    ${menu}    ${option_path[${path_len}-1]}    ${VALUE}
     RETURN    ${state}
+
+Get UEFI Boot Manager Entries
+    [Documentation]    Read list of UEFI boot manager
+
+    Power On
+
+    ${boot_menu}=    Enter Boot Menu Tianocore And Return Construction
```

Comment:
I'm a bit worried that the two `Get UEFI Boot Manager Entries` don't return the boot list in the same format, but it's probably fine for now.